src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,14 +1,46 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 /**
  * <p>Дано файл, потрібно прочитати його вміст і вибрати всі слова що починаються на `w`.
  * Результат повернути у вигляді відсортованого масиву (за замовчуванням). Всі слова повинні
  * бути в нижньому регістрі. У випадку якщо таких слів не знайдено повернути пустий масив.
  * Приклад: Width world Wide web
  * Результат: web wide width world</p>
  */
+
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+
+    private static final char LETTER = 'w';
+
+    public String[] readFromFile(String fileName) throws RuntimeException {
+
+        File file = new File(fileName);
+        ArrayList<String> textFromFile = new ArrayList<>();
```

Comment:
please review your variable names
this list is intended to store words that start with `w`

Reply:
renamed